pkg/ext-proc/server/runserver.go

```diff
@@ -23,24 +23,20 @@ type ExtProcServerRunner struct {
 	TargetEndpointKey      string
 	PoolName               string
 	PoolNamespace          string
-	ServiceName            string
-	Zone                   string
 	RefreshPodsInterval    time.Duration
 	RefreshMetricsInterval time.Duration
 	Scheme                 *runtime.Scheme
 	Config                 *rest.Config
 	Datastore              *backend.K8sDatastore
-	manager                ctrl.Manager
+	Manager                ctrl.Manager
```

Comment:
nit: Why exporting this?

Reply:
because we need to get the httpclient to create the custom informer